sda/cmd/s3inbox/proxy_test.go

```diff
@@ -451,11 +503,15 @@ func (suite *ProxyTests) TestDatabaseConnection() {
 	assert.NoError(suite.T(), err)
 	defer messenger.Connection.Close()
 	// Start proxy that allows everything
-	proxy := NewProxy(suite.S3conf, helper.NewAlwaysAllow(), messenger, database, new(tls.Config))
+	proxy := NewProxy(suite.S3Fakeconf, helper.NewAlwaysAllow(), messenger, database, new(tls.Config))
 
 	// PUT a file into the system
 	filename := "/dummy/db-test-file"
-	r, _ := http.NewRequest("PUT", filename, nil)
+
+	myString := "Hello, world!"
+	stringReader := strings.NewReader(myString)
+
+	r, _ := http.NewRequest("PUT", filename, stringReader)
```

Comment:
```suggestion
	stringReader := strings.NewReader("Hello, world!")
	r, _ := http.NewRequest("PUT", filename, stringReader)
```

Since the `myString` variable isn't going to be used it doesn't have to be declared, the string can be placed directly into the reader.